spec/features/saml/sp_initiated_slo_spec.rb

```diff
@@ -213,4 +213,34 @@
       expect(current_path).to eq('/')
     end
   end
+
+  context 'logged out of IDP' do
+    let(:user) { create(:user, :signed_up) }
+
+    context 'signed into one SP' do
+      before do
+        sign_in_and_2fa_user(user)
+        visit sp1_authnrequest
+
+        request = OneLogin::RubySaml::Logoutrequest.new
```

Comment:
That is probably a good idea. I'll leave it up to you if you want to do it in this PR or a new one.